docs/tutorial/create_footer_for_all_pages.md

```diff
@@ -122,6 +122,9 @@ Create an `includes` folder in your `mysite/templates` folder. Then in your newl
 Now, go to your `mysite/templates/base.html` file and modify it as follows:
 
 ```
+# add these two lines to avoid invalid block tags 
```

Comment:
No need to have this comment, I don't believe we do this for other tag imports. 

Other than that, change looks good. Thanks for fixing this.